Makefile

```diff
@@ -130,8 +130,9 @@ go-fmt:
 	go fmt $(PKGS)
 
 .PHONY: jsonnet-fmt
+export JSONNETFMT_BIN
 jsonnet-fmt: $(JSONNETFMT_BIN)
-	find jsonnet/ -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | xargs -n 1 -- $(JSONNETFMT_BIN) -i
```

Comment:
Why this needs a helper script? We already can execute `make generate` (which also downloads tooling) in CI without many problems as seen in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-monitoring-operator/openshift-cluster-monitoring-operator-master.yaml#L59-L62